### PR TITLE
Improves the CIN armor, adds some stuff, makes the surplus clothing part of the CIN surplus company sane for once

### PR DESCRIPTION
**Pull request by Paxilmaniac**
## About The Pull Request

The current CIN military armor is one of the few features I've added left that I still personally hate, the helmet was weird and the vest was just taken off of a warden jacket from somewhere. That, and, it didn't have anything special going for it.

So now, with the new component we have, lets be silly. The new armor covers your chest and arms for the vest, and your head of course with the helmet, and has 50 of both laser and bullet armor. This is pretty good, among the other stuff it gets, but like the sacrificial armor, it gets damaged when it protects you.

There is also a new belt type for use, one that instead of having seven slots for small items, has four slots for normal size items. Helpful if you want to carry ammo boxes or something in your belt.

## How This Contributes To The Skyrat Roleplay Experience

I don't like the stuff I made and think its insanely lame compared to what we have now, so I think we're better off when the stuff actually being sold has some unique or interesting thing to it.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

image
image
  
</details>

## Changelog

:cl:
add: There is a new chest rig alongside the reworked CIN stuff in cargo, holding four normal sized items, give it a try why don't you?
balance: The gz-03 vest and helmet have been reworked into the 'Klamra' armor set, armor that gives you good bullet and laser protection on your chest and limbs, or head for the helmet, in exchange for breaking if shot enough.
image: New sprites and resprites for various of the pieces of CIN equipment.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 1]
```diff
@@ -1413,6 +1423,16 @@ digi
...
 
+// Bigrig
+/datum/greyscale_config/cin_surplus_bigrig
+	name = "CIN Large Surplus Chestrig"
+	icon_file = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor.dmi'
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/nri_surplus_equipment/bigrig.json'
+
+/datum/greyscale_config/cin_surplus_bigrig/object
+	name = "CIN Large Surplus Chestrig (OBJ)"
+	icon_file = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor_object.dmi'
```
Comment (ghost) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]: Usually greyscale configs are the other way around. Base type is the obj, subtype is worn. There a reason these aren't?

Comment (Paxilmaniac) [reply modular_skyrat/modules/GAGS/greyscale_configs.dm]: That's just what I did for the other greyscale cin stuff and im not about to change it now

File: modular_skyrat/modules/company_imports/code/armament_datums/nri_military_surplus.dm [hunk 2]
```diff
@@ -2,64 +2,64 @@
...
-	item_type = /obj/item/clothing/suit/armor/vest/cin_surplus_vest
-	cost = PAYCHECK_COMMAND
+/datum/armament_entry/company_import/nri_surplus/clothing
+	subcategory = "Clothing Supplies"
+
+/datum/armament_entry/company_import/nri_surplus/clothing/uniform
+	item_type = /obj/item/clothing/under/syndicate/rus_army/cin_surplus/random_color
+	cost = PAYCHECK_CREW
 
 /datum/armament_entry/company_import/nri_surplus/clothing/police_uniform
```
Comment (Stalkeros2) [inline modular_skyrat/modules/company_imports/code/armament_datums/nri_military_surplus.dm:52]: (Remove the police gear entirely, I was only meaning for it to remain in Cargo for like what, a couple of weeks so people can use it for a while and then make it ghost role only.)

Comment (Stalkeros2) [reply modular_skyrat/modules/company_imports/code/armament_datums/nri_military_surplus.dm:52]: Or don't. I mean, who cares at this point.

Comment (Paxilmaniac) [reply modular_skyrat/modules/company_imports/code/armament_datums/nri_military_surplus.dm:52]: I am not going to

Comment (ghost) [reply modular_skyrat/modules/company_imports/code/armament_datums/nri_military_surplus.dm:52]: Please remove ERT/Antagonist faction gear from cargo.

File: modular_skyrat/modules/GAGS/greyscale_configs.dm (hunk 1 above)
Comment (vinylspiders) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
/datum/greyscale_config/cin_surplus_bigrig
	name = "CIN Large Surplus Chestrig"
	icon_file = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor_object.dmi'

/datum/greyscale_config/cin_surplus_bigrig/worn
	name = "CIN Large Surplus Chestrig (Worn)"
	icon_file = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor.dmi'
	json_config = 'modular_skyrat/modules/GAGS/json_configs/nri_surplus_equipment/bigrig.json'
```
Orion's right, these are named a certain way and while it technically doesn't matter it looks messy when looking through the greyscale UI.

File: modular_skyrat/modules/novaya_ert/code/surplus_armor.dm [hunk 3]
```diff
@@ -223,6 +254,46 @@
...
 	. = ..()
 
+/obj/item/storage/belt/military/big_cin_surplus
+	name = "'Ciężki' large chest rig"
+	desc = "The larger of the two common CIN chest rigs, able to hold four normal sized items."
+	icon = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor.dmi'
+	worn_icon = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor_object.dmi'
+	icon_state = "bigrig"
+	worn_icon_state = "bigrig"
+	greyscale_config = /datum/greyscale_config/cin_surplus_bigrig/object
```
Comment (vinylspiders) [inline modular_skyrat/modules/novaya_ert/code/surplus_armor.dm]:
```suggestion
	greyscale_config = /datum/greyscale_config/cin_surplus_bigrig
```

File: modular_skyrat/modules/novaya_ert/code/surplus_armor.dm [hunk 4]
```diff
@@ -223,6 +254,46 @@
...
+	greyscale_config_worn = /datum/greyscale_config/cin_surplus_bigrig
```
Comment (vinylspiders) [inline modular_skyrat/modules/novaya_ert/code/surplus_armor.dm]:
```suggestion
	greyscale_config_worn = /datum/greyscale_config/cin_surplus_bigrig/worn
```

File: modular_skyrat/modules/novaya_ert/code/surplus_armor.dm [hunk 5]
```diff
@@ -19,30 +19,37 @@
...
+	desc = "An outdated helmet used by CIN military forces. \
+		The design dates back to the years leading up to CIN - SolFed border war, and can still be seen today \
+		in some parts of CIN territories where its not so important for equipment to be the best of the best. \
+		While the armor will wear down over time, it covers both your arms and torso, and provides exceptional \
+		projectile and laser protection."
 	icon = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor.dmi'
 	worn_icon = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor_object.dmi'
 	icon_state = "helmet_plain"
 	greyscale_config = /datum/greyscale_config/cin_surplus_helmet/object
 	greyscale_config_worn = /datum/greyscale_config/cin_surplus_helmet
```
Comment (vinylspiders) [inline modular_skyrat/modules/novaya_ert/code/surplus_armor.dm]:
```suggestion
	greyscale_config = /datum/greyscale_config/cin_surplus_helmet
	greyscale_config_worn = /datum/greyscale_config/cin_surplus_helmet/worn
```

File: modular_skyrat/modules/novaya_ert/code/surplus_armor.dm [hunk 6]
```diff
@@ -148,34 +156,32 @@
...
+	desc = "An outdated armor vest used by CIN military forces. \
+		The design dates back to the years leading up to CIN - SolFed border war, and can still be seen today \
+		in some parts of CIN territories where its not so important for equipment to be the best of the best. \
+		While the armor will wear down over time, it covers both your arms and torso, and provides exceptional \
+		projectile and laser protection."
 	worn_icon = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor.dmi'
 	icon = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor_object.dmi'
 	icon_state = "vest_basic"
+	greyscale_config = /datum/greyscale_config/cin_surplus_vest/object
+	greyscale_config_worn = /datum/greyscale_config/cin_surplus_vest
```
Comment (vinylspiders) [inline modular_skyrat/modules/novaya_ert/code/surplus_armor.dm]:
```suggestion
	greyscale_config = /datum/greyscale_config/cin_surplus_vest
	greyscale_config_worn = /datum/greyscale_config/cin_surplus_vest/worn
```

File: modular_skyrat/modules/novaya_ert/code/surplus_armor.dm [hunk 7]
```diff
@@ -187,10 +193,35 @@
...
+	. = ..()
+
 // Chest Rig
 
 /obj/item/storage/belt/military/cin_surplus
-	desc = "A tactical webbing often used by the CIN's military forces."
+	name = "'Mały' chest rig"
+	desc = "The smaller of the two common CIN chest rigs, able to hold small items."
 	icon = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor.dmi'
 	worn_icon = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor_object.dmi'
```
Comment (vinylspiders) [inline modular_skyrat/modules/novaya_ert/code/surplus_armor.dm:226]:
```suggestion
	icon = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor_object.dmi'
	worn_icon = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor.dmi'
```
This is a mistake, right? Or are the icons actually in the wrong dmis? If that's the case you should fix that

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 8]
```diff
@@ -1380,6 +1380,16 @@ digi
...
 
+// Vest
+/datum/greyscale_config/cin_surplus_vest
+	name = "CIN Surplus Vest"
+	icon_file = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor.dmi'
+	json_config = 'modular_skyrat/modules/GAGS/json_configs/nri_surplus_equipment/vest.json'
+
+/datum/greyscale_config/cin_surplus_vest/object
+	name = "CIN Surplus Vest (OBJ)"
+	icon_file = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor_object.dmi'
```
Comment (vinylspiders) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]:
```suggestion
/datum/greyscale_config/cin_surplus_vest
	name = "CIN Surplus Vest"
	icon_file = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor_object.dmi'

/datum/greyscale_config/cin_surplus_vest/worn
	name = "CIN Surplus Vest (Worn)"
	icon_file = 'modular_skyrat/modules/novaya_ert/icons/surplus_armor/surplus_armor.dmi'
	json_config = 'modular_skyrat/modules/GAGS/json_configs/nri_surplus_equipment/vest.json'
```

**Review by vinylspiders — CHANGES_REQUESTED** [review]
I can't do code suggestions on the belt, helmet, and undersuit, but please do the same thing with those.

Also on a similar note make sure you update the `greyscale_config` and `greyscale_config_worn` for the remaining CIN items. (belt, undersuit, I think?)

**Review by vinylspiders — APPROVED** [review]
Thanks, looks good.

**Comment by vinylspiders** [comment]
@GoldenAlpharex sprites look fine to you?

**Pull request closed without merging**